From 83823733015998702e4dc0365764fe7dde4a321f Mon Sep 17 00:00:00 2001
From: Rui Miguel Silva <rui.silva@linaro.org>
Date: Wed, 1 Feb 2023 15:59:36 +0000
Subject: [PATCH 35/43] nvmxip: shorter block device name

Make the block device name shorter, so it will be set and presented
inside the array limits.

Upstream-Status: Pending
Signed-off-by: Rui Miguel Silva <rui.silva@linaro.org>
---
 drivers/nvmxip/nvmxip.c      | 2 +-
 drivers/nvmxip/nvmxip_qspi.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/nvmxip/nvmxip.c b/drivers/nvmxip/nvmxip.c
index af9c9a3b7270..91fe995f2d4d 100644
--- a/drivers/nvmxip/nvmxip.c
+++ b/drivers/nvmxip/nvmxip.c
@@ -91,7 +91,7 @@ int nvmxip_init(struct udevice *udev)
 
 	nvmxip_bdev_max_devs++;
 
-	snprintf(bdev_name, NVMXIP_BLKDEV_NAME_SZ, "nvmxip-blk#%d", nvmxip_bdev_max_devs);
+	snprintf(bdev_name, NVMXIP_BLKDEV_NAME_SZ, "blk#%d", nvmxip_bdev_max_devs);
 
 	ret = blk_create_devicef(udev, NVMXIP_BLKDRV_NAME, bdev_name, UCLASS_NVMXIP,
 				 nvmxip_bdev_max_devs, NVMXIP_DEFAULT_LBA_SZ,
diff --git a/drivers/nvmxip/nvmxip_qspi.c b/drivers/nvmxip/nvmxip_qspi.c
index 749625134acd..f6f5435e6377 100644
--- a/drivers/nvmxip/nvmxip_qspi.c
+++ b/drivers/nvmxip/nvmxip_qspi.c
@@ -43,7 +43,7 @@ static int nvmxip_qspi_of_to_plat(struct udevice *dev)
 		return -EINVAL;
 	}
 
-	pr_debug("[%s]: XIP device base addr: 0x%llx , lba_shift: %d , lbas: %lu\n",
+	log_err("[%s]: XIP device base addr: 0x%llx , lba_shift: %d , lbas: %lu\n",
 		 dev->name, plat_data->phys_base, plat_data->lba_shift, plat_data->lba);
 
 	return 0;
-- 
2.39.2

